src/main/java/io/vertx/core/http/HttpServerOptions.java

```diff
@@ -94,6 +94,11 @@
    */
   public static final boolean DEFAULT_DECOMPRESSION_SUPPORTED = false;
 
+  /**
+   * Default WebSocket Masked bit is true as depicted by RFC. PerformingUnMasking will be false
+   */
+  public static final boolean DEFAULT_FRAME_UNMASKING = false;
```

Comment:
last but not least rename `DEFAULT_FRAME_UNMASKING` to `DEFAULT_ACCEPT_UNMASKED_FRAMES`